our-initiatives/journal-club/journal-club-1.md

```diff
@@ -0,0 +1,17 @@
+---
+sidebar_position: 2
+---
+
+# Yves-Alexandre de Montjoye: The search for anonymous data 
+
+**Date: 9th October 2023**
+
+![Yves-Alexandre de Montjoye](../../static/img/journal-1-image.jpeg)
```

Comment:
Please create a directory `jc` under `img` and change the path to the following.
```suggestion
![Yves-Alexandre de Montjoye](/img/jc/journal-1-image.jpeg)
```